src/components/SideNav/Provider.js

```diff
@@ -1,58 +1,54 @@
 import React from "react";
 import PropTypes from "prop-types";
+import throttle from "lodash/throttle";
 
 import { DOM_TARGETS } from "constants/domNodes";
 
 import { findActiveNode } from "helpers/dom";
 
 const sortByPosition = (a, b) => {
-  const aY = a.current.getBoundingClientRect().top;
-  const bY = b.current.getBoundingClientRect().top;
+  const aY = a.ref.current.getBoundingClientRect().top;
+  const bY = b.ref.current.getBoundingClientRect().top;
   return aY - bY;
 };
 let lastScrollPosition = 0;
 
 export const SideNavProgressContext = React.createContext({});
 
 export const Provider = ({ children }) => {
-  const [activeNode, setActiveNode] = React.useState();
+  const [activeContent, setActiveNode] = React.useState({ id: "", ref: null });
```

Comment:
The rest of these changes are precipitated by changing `ScrollRouter` to own the SideNav provider as well. It's no longer sufficient to peel an ID off the inner child, so tracking needed to be updated to house a dom node + id